NorthstarDLL/masterserver.cpp

```diff
@@ -209,6 +210,78 @@ void MasterServerManager::RequestServerList()
 					goto REQUEST_END_CLEANUP;
 				}
 
+				// rapidjson conforms to JSON Schema draft 4
+				rapidjson::Document schemaDoc;
+				// this schema validates each entry in the array of servers
+				schemaDoc.Parse(
+					R"({
```

Comment:
Either indent the whole thing properly, or preferably start the object on a new line after the quote and keep the whole thing to the left.

Reply:
This was a clang formatting moment tbh

Reply:
That's what I meant by the second part (put the opening brace on a new line, but within the string literal so clang doesn't mangle it -- JSON doesn't care about whitespace).